pkg/git/git.go

```diff
@@ -263,3 +278,26 @@ func (g *Git) DeleteBranch(branch string) error {
 	}
 	return nil
 }
+
+// CheckFileExists checks if a file exists in the git repository for a specific branch
+func (g *Git) CheckFileExists(file, branch string) error {
+	upstreamRemote := g.Remotes["https://github.com/rancher/charts"]
+	target := upstreamRemote + "/" + branch + ":" + file
+
+	// Corrected command to only include the necessary arguments
+	cmd := exec.Command("git", "-C", g.Dir, "cat-file", "-e", target)
+	if err := cmd.Run(); err != nil {
+		return err // Return the error if the file does not exist or any other git error occurs
+	}
+	return nil // Return nil if the file exists
```

Comment:
I think we can instead directly return the `cmd.Run()` call.
```suggestion
	return exec.Command("git", "-C", g.Dir, "cat-file", "-e", target).Run()
```